msteams-platform/TOC.yml

```diff
@@ -416,110 +416,110 @@
 - name: Build bots
   items:
   - name: Overview
-    href: bots/what-are-bots.md
-  - name: Bots and SDKs
-    href: bots/bot-features.md
-    displayName: Bot samples, advantages of bots, bot with Power Virtual Agents, Microsoft Bot Framework
-  - name: Bot activity handlers
-    href: bots/bot-basics.md
-    displayName: OnMessageActivityAsync, Bot logic, core Bot Framework handlers
-  - name: Bot conversations
+    href: bots/overview.md
+    displayName: AI bots, bot capability, 
+  - name: Let's build a bot
```

Comment:
Instead of "Let's build a bot" (which sounds similar to "Build an AI bot" and "Let's build an AI bot" below), consider renaming this to "Decision guide" (or something a more specific to the article's content)